fundingmanager.go

```diff
@@ -1991,14 +2000,19 @@ func (f *fundingManager) handleFundingSigned(fmsg *fundingSignedMsg) {
 	// if we actually have the funding transaction.
 	if completeChan.ChanType.HasFundingTx() {
 		fundingTx := completeChan.FundingTxn
+		var fundingTxBuf bytes.Buffer
+		if err := fundingTx.Serialize(&fundingTxBuf); err != nil {
+			fndgLog.Errorf("Unable to serialize funding "+
+				"transaction %v: %v", fundingTx.TxHash(), err)
+		}
 
-		fndgLog.Infof("Broadcasting funding tx for ChannelPoint(%v): %v",
-			completeChan.FundingOutpoint, spew.Sdump(fundingTx))
+		fndgLog.Infof("Broadcasting funding tx for ChannelPoint(%v): %x",
+			completeChan.FundingOutpoint, fundingTxBuf.Bytes())
```

Comment:
Heh at once point I wrote a saved macro in my registry in vim to convert the spew dump log into actual Go syntax so I could rebroadcast a transaction... 

In this case, why not both? We could log the actual transaction (spew) on info, then the raw version on debug? Or the other way around makes more sense perhaps?

Reply:
There's been an effort to remove spew's other than `trace` so I went with this.

Reply:
I also prefer not to log the spew anymore. Also there shouldn't be any information in the spew that isn't serialized, right? Or is there some transient information that could be useful? Otherwise you can just run the serialized TX through `bitcoin-cli decoderawtransaction`.

Reply:
nit: Add the reset of the buffer here too?